packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -81,6 +81,8 @@ export type ParticipantListProps = {
   onParticipantClick?: (participant?: ParticipantListParticipant) => void;
   /** Styles for the {@link ParticipantList} */
   styles?: ParticipantListStyles;
+  /** True if using mobile */
+  isMobile?: boolean;
```

Comment:
We should always avoid isMobile flag on react-component (ideally it doesn't exist anywhere but we don't allow fine-grained customizations at the composite level).
Instead make this prop `showOverflowTooltip` (same with participant button only it will be slightly different and something like `showParticipantOverflowTooltip`)
The composite level can then do `showOverflowTooltip={props.isMobile}`

Reply:
For more context, the reason we avoid it is we want props to be extremely explicit about their intent and to only control one thing. If Contoso sets isMobile to true anything could happen inside the component

Reply:
Updated :)